posts/2021-10-01-ga-release-post.adoc

```diff
@@ -0,0 +1,170 @@
+
+== How to run Open Liberty on Java 17
+
+You will need a copy of Java 17.  To try this out, download Java 17 Temurin from link:https://adoptium.net/?variant=openjdk17[adoptium.net].
+
+Then you will need Open Liberty 21.0.0.10, or newer. To run your applications on Open Liberty 21.0.0.10, check out <<run, Run your apps using 21.0.0.10>>.
+
+There are a number of ways to control the Java executable used by the Open Liberty server. The following list describes the order of precedence:
+
+. `JAVA_HOME` in `${server.config.dir}/server.env`
+. `JAVA_HOME` in `${wlp.install.dir}/etc/server.env`
+. `JAVA_HOME` set in the environment
+. searching the `PATH` in the environment
+
+ For more information, please visit our link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html[server configuration documentation].
+Happy Java 17 experimenting!
+
+== Java 17 resources
+
+For further reading on Java 17, here are a few resources you might find useful:
+* Highlights of new features in Java 17 are found in the link:https://openjdk.java.net/projects/jdk/17/[Java 17 features page]
+* More detailed release information can be seen in the link:https://jdk.java.net/17/release-notes[release notes for Java 17]
+* Peruse link:https://docs.oracle.com/en/java/javase/17/docs/api/index.html[Javadocs for Java 17]
+
+For migration to Java 17, please check out link:https://blogs.oracle.com/javamagazine/migrate-to-java-17[Oracle Java 17 migration blog] and link:https://docs.oracle.com/en/java/javase/17/migrate/getting-started.html[Getting started with JDK 17].
+
+For Java 17 release roadmaps and availability refer to the link:https://adoptium.net/support.html?variant=openjdk17&jvmVariant=hotspot[Adoptium release roadmap] and link:https://www.oracle.com/java/technologies/java-se-support-roadmap.html[Oracle commercial JDK roadmap].
+
+And finally, for a comprehensive discussion on Java modules with a little hands on practice, read link:https://www.baeldung.com/java-9-modularity[this article from Baeldung].
+
+[#bugs]
+== Notable bugs fixed in this release
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues resolved in this release. If you’re interested, here’s the  link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A210010+label%3A%22release+bug%22[full list of bugs fixed in 21.0.0.10].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/18393[SPNEGO does not work with Java 11 or higher]
++
+In the release of Open Liberty 21.0.0.9, a regression was introduced which prevented the `Krb5LoginModule` class from being found when authenticating using Kerberos on Java 11 or higher. This issue caused some users to be presented with an `Error 403` or `Authentication Failed` when authenticating using the `spnego-1.0` feature. This issue has been fixed by reverting the underlying code back to the way it was prior to it being modified. 
```

Comment:
This issue was NOT fixed by reverting the underlying code.  The last sentence should say something like `This issue was fixed by changing how classes are loaded to work with Java 11 and hgiher releases.` or something like that.